src/commands/mdapi-deploy.yml

```diff
@@ -18,11 +18,16 @@ parameters:
     description: "Validate deploy, but don't save to org."
     type: boolean
     default: false
+  wait-time:
+    description: "Amount of time to wait for this to complete."
```

Comment:
Let's replace "this" with a more formal "Maximum amount of time to complete deployment" or something similar.